pkgs/tools/inputmethods/libpinyin/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl, glib, db, pkgconfig }:
+
+stdenv.mkDerivation {
+  name = "libpinyin-1.3.0";
+
+  meta = with stdenv.lib; {
+    description = "The libpinyin project aims to provide the algorithms core for intelligent sentence-based Chinese pinyin input methods.";
```

Comment:
This should be a rather short description. Maybe "Library for intelligent sentence-based Chinese pinyin input method"?